pandas/io/formats/style_render.py

```diff
@@ -989,6 +989,9 @@ def format(
             Use 'latex' to replace the characters ``&``, ``%``, ``$``, ``#``, ``_``,
             ``{``, ``}``, ``~``, ``^``, and ``\`` in the cell display string with
             LaTeX-safe sequences.
+            Use 'latex-math' to replace the characters ``&``, ``%``, ``#``, ``_``,
+            ``{``, ``}``, ``~``, ``^``, and ``\``
+            in the cell display string with LaTeX-safe sequences.
```

Comment:
Whilst this is true I think it is clearer to say:

"Replaces the same characters as the `latex` option **except** ``$`` which is preserved to allow conversion to math mode".

We should also add an example probably.